src/main/java/in/pune/royforge/eledgerapi/controller/WalletController.java

```diff
@@ -24,7 +24,7 @@ public void createWallet(@RequestBody WalletTransaction wallet) {
 	}
 
 	@RequestMapping(value = "/{walletId}", method = RequestMethod.GET)
-	public WalletData getWalletById(@PathVariable(value = "walletId") Long walletId) {
+	public WalletData getWallet(@PathVariable(value = "walletId") Long walletId) {
 		WalletData walletData = walletEntityService.getAWallet(walletId);
```

Comment:
please do the same as you do in service " return walletEntityService.getAWallet(walletId); "